src/lib/common/sem.h

```diff
@@ -65,6 +65,23 @@ extern int semInit
 
 /* ****************************************************************************
 *
+* reqSemTryToTake - try to take take semaphore
```

Comment:
Now that this only runs in DEBUG, I think the FIXME could be removed. In fact, we can replace that FIXME with:

"This method only runs in DEBUG compilations, so SIGSGV are not so critical"

Or a better wording explaining this thing